parts/kubernetesagentcustomdata.yml

```diff
@@ -113,7 +113,7 @@ write_files:
     DOCKER_OPTS=
     CUSTOM_CMD=/bin/true
     KUBELET_REGISTER_SCHEDULABLE=true
-    KUBELET_NODE_LABELS=role=agent
+    KUBELET_NODE_LABELS=role=agent,failure-domain.beta.kubernetes.io/region={{WrapAsVariable "location"}}
```

Comment:
There are two labels associated with Fault Domain (FD): failure-domain.beta.kubernetes.io/region and failure-domain.beta.kubernetes.io/zone